posdealers/03-sales/04c-masterdata-creation.md

```diff
@@ -117,8 +117,13 @@ Nachfolgend wird beschreiben, welche Rolle bezüglich der Verwaltung der Stammda
 
 ## Aktualisierung und Wirksamkeit der Stammdaten
 
-Eine Aktualisierung der oben beschriebenen Daten wird erst wirksam, wenn die betroffenen Cashboxen des KassenBetreibers neu gebuildet ("rebuild configuration") und die launcher an den Kassen neu gestartet wurden, damit diese die neuen Cashbox-Konfigurationen laden und anwenden können.
+Eine Aktualisierung der oben beschriebenen Daten wird erst wirksam, wenn die betroffenen Cashboxen des KassenBetreibers neu gebuildet ("rebuild configuration") und die Launcher an den Kassen neu gestartet wurden, damit diese die neuen Cashbox-Konfigurationen laden und anwenden können. Zudem ist ein Wechsel der Stammdaten nur jeweils zum Kassenabschluss möglich, d.h. die neuen Daten werden erst zu diesem Zeipunkt gültig.
 
-### vertiefende Informationen
+Zusammengefasst bedeutet das, das folgende Schritte durchgeführt werden müssen:
+1. Die nötigen Änderungen der Stammdaten (z.B. Agenturdaten, POS-Systeme, etc.) werden im Portal durchgeführt.
+2. Die Cashbox wird rebuildet und die Middleware danach neu gestartet, damit die lokale Konfiguration aktualisiert werden kann. **Danach werden die neuen Stammdaten nicht sofort gültig, sondern werden bis zum nächsten Daily-Closing-Beleg mit dem entsprechenden Flag ignoriert.**
+3. Ein Daily-Closing-Beleg mit dem _ftReceiptCaseFlag_ `0x0000000008000000` wird gesendet. Bis zu diesem Zeitpunkt wird die alte Konfiguration verwendet, d.h. die Aktualisierung der Stammdaten scheint im DSFinV-K-Exports dieses zum Zeitpunkt der Änderung noch laufenden Kassenabschlusses nicht auf. Erst im Export des darauffolgenden Abschlusses sind die neuen Daten gültig.
```

Comment:
```suggestion
3. Ein Kassenabschlussbeleg mit Stammdatenupdate wird von der Kasse an den fiskaltrust.Service gesendet. Ihr Kassenhersteller sollte diese Funktion über einen sogenannten Daily-Closing-Beleg mit dem _ftReceiptCaseFlag_ [`0x0000000008000000`](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/germany/reference-tables/ftreceiptcase#ftreceiptcaseflag) ins Kassensystem integriert haben. Die Aktualisierung der Stammdaten wird erst nach dem Kassenabschlussbelegs mit Stammdatenupdate aktiviert und die neuen Daten erscheinen damit erst im DSFinV-K-Export des nächsten, darauffolgenden Abschlusses.
```

Reply:
Thanks! I applied @AxelKutschera's suggestion, as it contained the (correct) information that this is also working with monthly- or yearly-closing receipts :)

Reply:
Thanks, @AxelKutschera suggestion is good, but I would also try to update the part with the flag because I have the feeling that POSDealers can not use this information since only the POSCreator can apply the flag during the integration. The POSDealer should have a button in POSSystem to do so

Reply:
I now rephrased the sentence to also include your suggestions